src/libstd/collections/hash/map.rs

```diff
@@ -1282,7 +1291,7 @@ pub struct IterMut<'a, K: 'a, V: 'a> {
 /// HashMap move iterator.
 #[stable(feature = "rust1", since = "1.0.0")]
 pub struct IntoIter<K, V> {
-    inner: table::IntoIter<K, V>,
+    pub(super) inner: table::IntoIter<K, V>,
```

Comment:
Necessary because `HashSet` uses `HashMap` internally and it needs access to this `inner` field to get the underlying entries without mutating.